docs/index.rst

```diff
@@ -114,6 +114,14 @@
 .. toctree::
   :maxdepth: 0
   :hidden:
+  :caption: Stackdriver Error Reporting
```

Comment:
To clarify, Monitoring is fine as a short name. In this Toc it says "Cloud Logging" and "Cloud Monitoring" which should be changed.